src/io/flutter/actions/FlutterSdkAction.java

```diff
@@ -49,15 +50,25 @@
     return pubspec == null ? null : Pair.create(module, pubspec);
   }
 
-  protected static VirtualFile findPubspecFrom(Project project, PsiFile psiFile) {
+  protected static VirtualFile findPubspecFrom(@NotNull Project project, PsiFile psiFile) {
     if (psiFile == null) {
-      return null;
+      return findPubspecFrom(ModuleManager.getInstance(project).getModules());
     }
     final VirtualFile file = psiFile.getVirtualFile();
     final VirtualFile contentRoot = ProjectRootManager.getInstance(project).getFileIndex().getContentRootForFile(file);
     return contentRoot == null ? null : contentRoot.findChild(FlutterConstants.PUBSPEC_YAML);
   }
 
+  private static VirtualFile findPubspecFrom(@NotNull Module[] modules) {
+    for (Module module : modules) {
+      final VirtualFile file = findPubspecFrom(module);
```

Comment:
If there's only one match, returning that makes sense. If there's more than one, we may be better off failing here than running the command for a different module than expected.